Examples/Embedded/Sources/EmbeddedApp/_thingsThatShouldNotBeNeeded.swift

```diff
@@ -0,0 +1,29 @@
+import JavaScriptKit
+
+// NOTE: it seems the embedded tree shaker gets rid of these exports if they are not used somewhere
+func _i_need_to_be_here_for_wasm_exports_to_work() {
```

Comment:
Could you try adding `@_used` (for Swift decls) or `__attribute__((used))` (for C decls) to teach LTO not to DCE those decls?

Reply:
I tried `@_used` for both `@_expose` and `@_cdecl`, does not work : (
I'll post in the forums about it, maybe this can be fixed in the future somehow.
An underscored public is is the best I can come up with atm.